imports.go

```diff
@@ -125,22 +125,22 @@ func tryPath(dir, importedPath string) (found bool, content []byte, foundHere st
 func (importer *FileImporter) Import(dir, importedPath string) *ImportedData {
 	found, content, foundHere, err := tryPath(dir, importedPath)
 	if err != nil {
-		return &ImportedData{err: err}
+		return &ImportedData{Err: err}
```

Comment:
- [ ] Did you consider instead making the function signature return `(*ImportedData, error)` and removing the `Err` field from `ImportedData`?  
  That is, `Import` either returns imported data or an error. When returning an error, the `*ImportedData` value would be nil.

Reply:
IIRC I originally put `err` inside, because I wanted errors to be cached. But I agree this feels a bit unnatural.

Reply:
Have done

Reply:
Nice. 👍